packages/blocks/src/class-blocks.php

```diff
@@ -17,7 +17,59 @@
  * @since 9.0.0
  */
 class Blocks {
+	/**
+	 * Get CSS classes for a block.
+	 *
+	 * @since 9.0.0
+	 *
+	 * @param string $slug  Block slug.
+	 * @param array  $attr  Block attributes.
+	 * @param array  $extra Potential extra classes you may want to provide.
+	 *
+	 * @return string $classes List of CSS classes for a block.
+	 */
+	public static function classes( $slug = '', $attr, $extra = array() ) {
+		if ( empty( $slug ) ) {
+			return '';
+		}
 
+		// Basic block name class.
+		$classes = array(
+			'wp-block-jetpack-' . $slug,
+		);
+
+		// Add alignment if provided.
+		if (
+			! empty( $attr['align'] )
+			&& in_array( $attr['align'], array( 'left', 'center', 'right', 'wide', 'full' ), true )
+		) {
+			array_push( $classes, 'align' . $attr['align'] );
```

Comment:
Why `array_push()` instead of
```
$classes[] = 'align' . $attr['align'];
```
?

Reply:
No specific reason :) 

Updated in f50d2d3494c677568a1e93d97bf425b335f6e83b